modules/translation/translation.js

```diff
@@ -23,20 +23,7 @@ class Translation {
     /**
      *
      */
-    generateTranslationHTML(key: string, options: Object) {
-        const optAttr
-            = options ? ` data-i18n-options='${JSON.stringify(options)}'` : '';
-
-        // XXX i18next expects undefined if options are missing.
-        const text = i18next.t(key, options ? options : undefined);
-
-        return `<span data-i18n="${key}"${optAttr}>${text}</span>`;
-    }
-
-    /**
-     *
-     */
-    init() {
```

Comment:
Do you happen to know why this had a separate init method?

Reply:
I don't remember but it seems to work with the constructor.